src/Shared/Model/ParticipantScreeningEpisode.cs

```diff
@@ -5,6 +5,8 @@ namespace NHS.ServiceInsights.Model;
 
 public partial class ParticipantScreeningEpisode
 {
+    public int Id { get; set; }
```

Comment:
I think you might need to re-scaffold the entity models since I would expect this to be long